CRM/Contribute/Page/ContributionRecur.php

```diff
@@ -61,8 +61,20 @@ public function view() {
       if (!empty($values['contribution_status_id'])) {
         $values['contribution_status'] = CRM_Core_OptionGroup::getLabel('contribution_status', $values['contribution_status_id']);
       }
+      // Get financial type name
+      if (!empty($values['financial_type_id'])) {
+        $values['financial_type_name'] = CRM_Contribute_PseudoConstant::financialType($values['financial_type_id']);
+      }
+      // Get Paid By label
+      if (!empty($values['payment_instrument_id'])) {
+        $values['payment_instrument'] = CRM_Core_OptionGroup::getLabel('payment_instrument', $values['payment_instrument_id']);
```

Comment:
cool!

On 30/10/2015 1:50 a.m., Rajesh Sundararajan wrote:
>
> In CRM/Contribute/Page/ContributionRecur.php 
> <https://github.com/civicrm/civicrm-core/pull/6291#discussion_r43380151>:
>
> > @@ -61,8 +61,20 @@ public function view() {
> >        if (!empty($values['contribution_status_id'])) {
> >          $values['contribution_status'] = CRM_Core_OptionGroup::getLabel('contribution_status', $values['contribution_status_id']);
> >        }
> > +      // Get financial type name
> > +      if (!empty($values['financial_type_id'])) {
> > +        $values['financial_type_name'] = CRM_Contribute_PseudoConstant::financialType($values['financial_type_id']);
> > +      }
> > +      // Get Paid By label
> > +      if (!empty($values['payment_instrument_id'])) {
> > +        $values['payment_instrument'] = CRM_Core_OptionGroup::getLabel('payment_instrument', $values['payment_instrument_id']);
>
> Updated the code to use cached function
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/civicrm/civicrm-core/pull/6291/files#r43380151>.
>



---
This email has been checked for viruses by Avast antivirus software.
https://www.avast.com/antivirus